### PR TITLE
Added Qdrants overview.md

### PR DESCRIPTION
**Pull request by jackquix**
## Description

Adding overview sections for Qdrant, Weaviate, Upstash, and a broader overview for the Vector Database subsection. At the end I will add mkdocks.yml

## Review

Proofread the text and check the links, finally check that mkdocs.yml file embeds correctly.

### CODE REVIEW

File: docs/integrations/vector-databases/qdrant/overview.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+title: Overview - Qdrant
+description: This section of the Quix documentation covers how to integrate QDrant with Quix.
+search:
```
Comment (tbedford) [inline docs/integrations/vector-databases/qdrant/overview.md]: This isn't required.

File: docs/integrations/vector-databases/overview.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+title: Overview - Vector Databases
+description: A summary of what vector databases are.
+search:
```
Comment (tbedford) [inline docs/integrations/vector-databases/overview.md]: This isn't required.

File: docs/integrations/vector-databases/qdrant/overview.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  boost: 3
+---
+
+Qdrant is a vector similarity search engine and vector database written in Rust. It enables storage, searching and management of points with associated payloads. Qdrant supports filtering, sparse vectors, and distributed deployment for various AI use cases.
+
+A template sink connector for Qdrant VectorDB can be found [here](https://github.com/quixio/template-continuous-vector-ingestion/tree/main/Ingest%20to%20Qdrant%20VectorDB). There is also a connector for Qdrant Cloud VectorDB, which is available [here](https://github.com/quixio/template-continuous-vector-ingestion/tree/main/Ingest%20to%20Qdrant%20Cloud%20VectorDB).
```
Comment (tbedford) [inline docs/integrations/vector-databases/qdrant/overview.md]: External links should always use `{target=_blank}`.

**Review by tbedford — CHANGES_REQUESTED** [review]
* The PR description says "Qdrant, Weaviate, Upstash", but I only see Qdrant. 
* The `mkdocs.yml` file hasn;t been updated to include this, so they currently do not appear in the nav.

**Comment by jackquix** [comment]
> * The PR description says "Qdrant, Weaviate, Upstash", but I only see Qdrant.
> 
>     * The `mkdocs.yml` file hasn;t been updated to include this, so they currently do not appear in the nav.

I'm commiting one document at a time and I'll submit the PR once they're all added to the PR. Feel free to help me improve this work flow

File: docs/integrations/vector-databases/qdrant/overview.md (hunk 1 above)
Comment (jackquix) [reply docs/integrations/vector-databases/qdrant/overview.md]: Why isn't it? What does this do?

Comment (tbedford) [reply docs/integrations/vector-databases/qdrant/overview.md]: It causes the page to rank higher in search, which is not required for this page. It's only used for very specific pages we want to boost.

Comment (tbedford) [reply docs/integrations/vector-databases/qdrant/overview.md]: Also, I just notice you have two capitalizations of Qdrant - in the description you use "QDrant" which is not correct from brief check of their website.

**Comment by tbedford** [comment]
> I'm commiting one document at a time and I'll submit the PR once they're all added to the PR. Feel free to help me improve this work flow

The normal workflow would be to add a reviewer when you're happy with the PR. This saves time, as there's no point getting a review done if you're then going to add more files, as the review would need to be done again. Thanks.

**Comment by jackquix** [comment]
> > I'm commiting one document at a time and I'll submit the PR once they're all added to the PR. Feel free to help me improve this work flow
> 
> The normal workflow would be to add a reviewer when you're happy with the PR. This saves time, as there's no point getting a review done if you're then going to add more files, as the review would need to be done again. Thanks.

But the work flow of commit commit commit to a PR is okay?

**Comment by tbedford** [comment]
> But the work flow of commit commit commit to a PR is okay?

Yes.

**Comment by jackquix** [comment]
I expect the Upstash and Weaviate overviews to be updated when the people from those companies get back to me. Hopefully early next week. Otherwise I will just merge

File: docs/integrations/vector-databases/overview.md (hunk 2 above)
Comment (tbedford) [reply docs/integrations/vector-databases/overview.md]: search metadata is still here.

File: docs/integrations/vector-databases/upstash/overview.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+---
+title: Overview - Upstash
+description: This section of the Quix documentation covers how to integrate Upstash with Quix.
+---
```
Comment (tbedford) [inline docs/integrations/vector-databases/upstash/overview.md:4]: You need a blank line after the `---` and the heading.

File: docs/integrations/vector-databases/overview.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+  boost: 3
+---
+
+## Overview - Vector Databases
```
Comment (tbedford) [inline docs/integrations/vector-databases/overview.md]: For some reason you've done the headings here as `##` they should be `#` for the top level heading.

File: mkdocs.yml [hunk 6]
```diff
@@ -165,6 +165,14 @@ nav:
         - 'Overview': 'integrations/databases/postgressql/overview.md'
       - 'Redis':
         - 'Overview': 'integrations/databases/redis/overview.md'
+    - 'Vector Databases':
+      - 'Overview': 'integrations/vector-databases/overview.md'
+      - 'Upstash':
+        - 'integrations/vector-databases/qdrant/overview.md'
+      - 'Qdrant':
+        - 'docs/integrations/vector-databases/qdrant/overview.md'
```
Comment (tbedford) [inline mkdocs.yml]: The link here is wrong:

'docs/integrations/vector-databases/qdrant/overview.md'

It should be 

'integrations/vector-databases/qdrant/overview.md'

File: mkdocs.yml [hunk 7]
```diff
@@ -165,6 +165,14 @@ nav:
         - 'Overview': 'integrations/databases/postgressql/overview.md'
       - 'Redis':
         - 'Overview': 'integrations/databases/redis/overview.md'
+    - 'Vector Databases':
+      - 'Overview': 'integrations/vector-databases/overview.md'
+      - 'Upstash':
```
Comment (tbedford) [inline mkdocs.yml:170]: This is wrong, it's the entry for Upstash but you link to the Qdrant file.

**Review by tbedford — CHANGES_REQUESTED** [review]
Some serious issues. Please do not merge. 

See comments, also screenshot:

image

**Review by tbedford — APPROVED** [review]
LGTM

**Pull request merged**